packages/ui/src/primitives/tailwind/Checkbox/index.tsx

```diff
@@ -32,7 +32,7 @@ import Label from '../Label'
 
 export interface CheckboxProps extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'value' | 'onChange'> {
   value: boolean
-  label?: string
+  label?: string | React.ReactNode
```

Comment:
If label is a ReactNode, it should be rendered as it is at Line 63 (not inside the Label component).

Reply:
also type of label will be `label?: React.ReactNode` only since string is a sub type

Reply:
Yes, we should use `label?: React.ReactNode` instead of `label?: string | React.ReactNode`